string plugin_name
string[] agents
string[] map_names

---
bool result